bucket/kando.json

```
@@ -0,0 +1,21 @@
+{
+    "version": "1.1.0",
+    "description": "The Cross-Platform Pie Menu.",
+    "homepage": "https://github.com/kando-menu/kando",
+    "license": "MIT",
+    "url": "https://github.com/kando-menu/kando/releases/download/v1.1.0/Kando-win32-x64-1.1.0.zip",
+    "hash": "04a85258ef1d068aa7bbfd2ff661b45f5e13a42d918856a5d7f696e7fc51e9eb",
+    "bin": "kando.exe",
+    "shortcuts": [
+        [
+            "kando.exe",
+            "Kando"
+        ]
+    ],
+    "checkver": {
+        "github": "https://github.com/kando-menu/kando"
+    },
```

```suggestion
    "checkver": "github",
```

Since same GH link already used as `homepage`.